client/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Google</title>
+    <link rel="stylesheet" href="./style.css" />
+    <script defer src="./bundle.js"></script>
+  </head>
+  <body class="primary">
+    <section>
+      <header>
+        <div class="links">links</div>
```

Comment:
what's this for?

Reply:
Was originally going to use flex to space the contents of the section tag but changed plan and forgot to removes section.